ui/src/message_popup/author/questions_page.jsx

```diff
@@ -0,0 +1,162 @@
+/* @flow weak */
+import React from 'react';
+import _ from 'lodash';
+
+import * as Routes from '../../routes.js';
+
+import TextField from 'material-ui/TextField';
+import Divider from 'material-ui/Divider';
+import IconButton from 'material-ui/IconButton';
+import Paper from 'material-ui/Paper';
+import MenuItem from 'material-ui/MenuItem';
+import {Card, CardHeader, CardText} from 'material-ui/Card';
+
+import SearchIcon from 'material-ui/svg-icons/action/search';
+import AddIcon from 'material-ui/svg-icons/content/add';
+import ArchiveIcon from 'material-ui/svg-icons/content/archive';
+
+import NavigationAppBar from '../../components/navigation_app_bar.jsx';
+import QuestionButton from './question_button.jsx';
+
+import {allQuestions} from '../questions.js';
+import {withStudents, withLearningObjectiveAndIndicator} from '../transformations.jsx';
+
+export default React.createClass({
+  displayName: 'QuestionsPage',
+
+  getInitialState(){
+    return ({
+      questions: allQuestions,
+      showArchivedQuestions: false
+    });
+  },
+  
+  onNewQuestion(){
+    Routes.navigate(Routes.messagePopupAuthorQuestionsNewPath());
+  },
+
+  onArchivedQuestions(){
+    Routes.navigate(Routes.messagePopupAuthorArchivedQuestionsPath());
+  },
+
+  onSearchBarChange(event){
+    const value = event.target.value.toLowerCase().trim();
```

Comment:
Consider what happens when the user enters search text with a space, which I think might be common in this case.  What would the user expect to happen?